### DISCUSSION TITLE
TTL filer deleted,but sync filer still exists

### DISCUSSION THREAD

**Discussion by LuuucasChen** [seaweedfs/seaweedfs#4712] [Q&A]
I found a bug. I used the "ttl" parameter of the filer to upload a file and simultaneously started the filer.sync process. When the source file is deleted due to expiration, no relevant logs are generated, and the synchronized replicas are not deleted either.

**Comment by chrislusf** [comment]
TTL, same as replication and diskType, is determined by the destination filer settings.

Other types of destinations usually do not have TTL support.

**Reply by LuuucasChen** [reply]
Yes, and metadata like "collection" cannot be synchronized either.
